shotgun_api3/lib/mockgun.py

```diff
@@ -572,6 +572,19 @@ def _compare(self, field_type, lval, operator, rval):
                 return lval == rval
             elif operator == "is_not":
                 return lval != rval
+
+        elif (field_type in ("date", "date_time") and
+                      operator in ('in_calendar_day', 'in_calendar_week', 'in_calendar_month')):
+            today = datetime.date.today()
+            current_year, current_week, current_day = today.isocalendar()
+            year, week, day = lval.isocalendar()
+            if operator == "in_calendar_day":
```

Comment:
I think this operator is not obvious to understand compared to others and maybe it should warrant an explanation of what it does.